### DISCUSSION TITLE
Run jobs on multiple databases

### DISCUSSION THREAD

**Discussion by starlessboi** [cybertec-postgresql/pg_timetable#407] [Q&A] [answered]
Hi
I am a newbie here, thanks for your amazing job

In my scenario, I have a database server with multiple databases, I need to run jobs and queries on schedule over those databases.
what i got from your documents, i ran a pg_timetable docker in a remote server and made a connection to my database, created a db named timetable, upon running service, some tables created and everything worked normally. then created some jobs with add_job command according to examples successfully.
here is my question, how I can run some queries on other databases in my database servers?  for example select * from somedb.public.tablename
I created a job with above logic, it says: ERROR: cross-database references are not implemented
should I use something like dblink...?
how can I run this query straight on target database?
I think, may be I missed some basic concepts... would be happy If give some information over this topic and tell me best practices
Thanks in advance

**Comment by pashagolub** [comment, marked as answer]
Hi. Thanks for your interest.

To run any task against a remote database you should use `database_connection` field in the `timetable.task` table. Find more details [here](https://pg-timetable.readthedocs.io/en/master/components.html#table-timetable-task).

Also we have a `RemoteDB` [example](https://github.com/cybertec-postgresql/pg_timetable/blob/master/samples/RemoteDB.sql) in the `samples` folder.

Have a nice day!

**Reply by starlessboi** [reply]
with some modifications, it works like a charm
Thank You
